WTWSMS/common/landed_titles/WtWSMS_holy_orders.txt

```diff
@@ -29,7 +29,7 @@ d_agonisticis = {
 	mercenary_type = teutonic_order_composition
 }
 
-d_sunni_berber_company = { 
+d_valentinian_brotherhood = { 
```

Comment:
This is an intentional reutilisation of the vanilla content which is overwritten elsewhere.